browser/pages/account/account.html

```diff
@@ -14,20 +14,30 @@
           <input type="text" id="username" name="username" class="login-form-input form-control" ng-model="acctCtrl.username" ng-disabled="acctCtrl.isLoginBtnClicked" required aria-describedby="usernameStatus" autofocus/>
           <!--<span class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true" ng-show="(accountForm.$submitted || accountForm.username.$touched) && accountForm.username.$invalid"></span>-->
           <span id="usernameStatus" class="sr-only" ng-show="(accountForm.$submitted || accountForm.username.$touched) && accountForm.username.$invalid">(error)</span>
-          <div ng-show="(accountForm.$submitted || accountForm.username.$touched) && accountForm.username.$error.required" class="align-to-input help-block has-error">Please enter a username.</div>
         </div>
         <div class="login-form-input-container form-group has-feedback" ng-class="{'has-error has-feedback': (accountForm.$submitted || accountForm.password.$touched) && accountForm.password.$invalid}">
           <label for="password" class="login-form-label">Password</label>
           <input type="password" id="password" name="password" class="login-form-input form-control" ng-model="acctCtrl.password" ng-disabled="acctCtrl.isLoginBtnClicked" required aria-describedby="passwordStatus"/>
           <!--<span class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true" ng-show="(accountForm.$submitted || accountForm.password.$touched) && accountForm.password.$invalid"></span>-->
           <span id="passwordStatus" class="sr-only" ng-show="(accountForm.$submitted || accountForm.password.$touched) && accountForm.password.$invalid">(error)</span>
-          <div ng-show="(accountForm.$submitted || accountForm.password.$touched) && accountForm.password.$error.required" class="align-to-input help-block has-error">Please enter a password.</div>
         </div>
-        <div ng-show="accountForm.$submitted && (acctCtrl.authFailed || acctCtrl.tandcNotSigned || acctCtrl.httpError)" class="form-group has-error pull-right">
-          <div id="invalidLoginError" ng-show="accountForm.$submitted && acctCtrl.authFailed" class="help-block">
+        <div ng-show="(accountForm.$submitted || accountForm.password.$touched) && ((acctCtrl.authFailed || acctCtrl.tandcNotSigned || acctCtrl.httpError) || (accountForm.username.$error.required || accountForm.password.$error.required))" class="form-group has-error pull-right">
```

Comment:
ng-show is not needed here, because if divs below is not visible, there would be no text shown anyway